tests/lib/rules/jsx-no-literals.js

```diff
@@ -127,6 +127,48 @@ ruleTester.run('jsx-no-literals', rule, {
       parser: 'babel-eslint',
       options: [{noStrings: true}]
     }, {
+      code: '<Foo bar={true} />',
+      parser: 'babel-eslint',
+      options: [{noStrings: true}]
+    }, {
+      code: '<Foo bar={false} />',
+      parser: 'babel-eslint',
+      options: [{noStrings: true}]
+    }, {
+      code: '<Foo bar={100} />',
+      parser: 'babel-eslint',
+      options: [{noStrings: true}]
+    }, {
+      code: '<Foo bar={null} />',
+      parser: 'babel-eslint',
+      options: [{noStrings: true}]
+    }, {
+      code: '<Foo bar={{}} />',
+      parser: 'babel-eslint',
```

Comment:
babel-eslint isn't needed for any of these new tests; let's please prefer the default parser whenever possible.